File: gtServer/Objects/GuitarFretboard.cs
namespace gtServer.Objects;

public class GuitarFretboard
{
    public List<GuitarString> GuitarStrings { get; set; }
    int NumStrings { get; set; }

    public GuitarFretboard(int numStrings = 6)
    {
        NumStrings = numStrings;
        List<GuitarString> strings = [];
        for (int i = 0; i < numStrings; i++)
        {
            GuitarString freshString = new();
            strings.Add(freshString);
        }
        GuitarStrings = strings;
    }
}

File: fastapi-server/objects/guitar_fretboard.py
from objects.guitar_string import GuitarString


class GuitarFretboard():
    def __init__(self, num_strings: int):
        self.num_strings: int = num_strings
        self.strings: list[GuitarString] = []

        self.string_guitar()

    def string_guitar(self):
        self.strings = []
        for index in range(0, self.num_strings):
            self.strings.append(GuitarString())
